src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -450,6 +450,9 @@ bool emberAfOperationalCredentialsClusterAddNOCCallback(app::CommandHandler * co
     err = Server::GetInstance().GetFabricTable().Store(fabricIndex);
     VerifyOrExit(err == CHIP_NO_ERROR, nocResponse = ConvertToNOCResponseStatus(err));
 
+    // Notify the secure session of the new fabric.
+    commandObj->GetExchangeContext()->GetSessionHandle()->AsSecureSession()->NewFabric(commandObj->GetAccessingFabricIndex());
```

Comment:
Shouldn't this pass `fabricIndex`, not `commandObj->GetAccessingFabricIndex()`?  I don't see how `commandObj->GetAccessingFabricIndex()` would return anything useful here....

Reply:
Good catch, I did all my testing with correct code, then when I removed log statements and simplified it to one line I screwed it up by copy pasting that line from another function.